extensions/wikia/Chat2/ChatBanListSpecial_helper.php

```diff
@@ -0,0 +1,269 @@
+<?php
+
+/**
+ * @package MediaWiki
+ * @subpackage SpecialPage
+ * @author Piotr Molski <moli@wikia.com>
+ * @version: $Id$
+ */
+
+use Wikia\Service\User\Permissions\PermissionsServiceAccessor;
+
+class ChatBanData
```

Comment:
Let's extend `WikiaModel` here. It'll give access to DB helpers and Nirvana framework vars.